lib/modifiers.js

```diff
@@ -45,7 +45,20 @@ function nestedRestore (arr) {
   const length = arr.length
   for (var i = 0; i < length; i++) {
     const { key, target, value } = arr[i]
-    target[key] = value
+    if (has(target, key)) {
+      target[key] = value
+    }
+    /* istanbul ignore else */
+    if (typeof target === 'object') {
```

Comment:
Is it possible that target is not an object? It's not covered by tests anyway

Reply:
I think its possible, applies for the example below:

```js
const redact = fastRedact({
  paths: ['a[*].x.d[*].i.*.i']
})
const o = {
  a: [
    { x: { d: [ { i: undefined, j: 'NR' } ] } }
  ]
}
```

Only applies when the given path expects a deeper nested object but is undefined somewhere.